index.html

```diff
@@ -4,26 +4,29 @@
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Microverse Portfolio</title>
-  <link rel="stylsheet" href="style.css">
+  <title>Microverse portfolio</title>
+  <link rel= "stylesheet" href="style.css"/>
 </head>
 
 <body>
-  <header class="header">
-    <nav class="menu">☰ </nav>
-  </header>
-  <main>
-    <section class="Home">
-      <h1>Sonia Bangar</h1>
-      <p class="paragraph">Hello! I am a software developer! I can help you build a product, feature or website. Taka a
-        look of my works.don&apos;t
-        If you like what you see and have a project you need coded, hesitate and contact me.</p>
-      <button class="start">Start collaboration</button>
-
+    <div class= "toolbar">
+      <div class= "img_geomtry">
+        <img src= "./images/image_geometry_1.png" alt=""class="shape1">
+      </div>
+      <nav class= "navbar">
```

Comment:
- [ ] Please make sure that the container of the section is a `header` tag. typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.